packages/edit-site/src/components/secondary-sidebar/list-view-sidebar.js

```diff
@@ -59,7 +64,9 @@ export default function ListViewSidebar() {
 					focusOnMountRef,
 				] ) }
 			>
-				<ListView />
+				sdfds
+				<ListView LeafMoreMenu={ LeafMoreMenu } />
```

Comment:
All the changes in this file are simply to show that it works, we'll revert before merging.